File: android/src/main/java/expo/modules/bluetoothprinter/data/BluetoothPrinterValue.kt
package expo.modules.bluetoothprinter.data

data class BluetoothPrinterValue(
  val image: BluetoothPrinterImage?,
  val text: BluetoothPrinterText?,
  val qrCode: BluetoothPrinterQRCode?,
  val newLines: Int?,
)

File: src/data/BluetoothPrinterValue.ts
import { BluetoothPrinterText } from "./BluetoothPrinterText";
import { BluetoothPrinterImage } from "./BluetoothPrinterImage";
import { BluetoothPrinterQRCode } from "./BluetoothPrinterQRCode";

export type BluetoothPrinterValue = {
  image?: BluetoothPrinterImage;
  text?: BluetoothPrinterText;
  qrCode?: BluetoothPrinterQRCode;
  newLines?: number;
};
